packages/api/jest.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   moduleFileExtensions: ['js', 'json', 'ts', 'node', 'd.ts'],
   globals: {
     'ts-jest': {
-      diagnostics: true,
+      diagnostics: false,
```

Comment:
We could limit diagnostics here to only exclude the "TS2722" code. This way we wont miss other, possibly helpful diagnostics:

`diagnostics: {
ignoreCodes: 'TS2722'
}
`

Reply:
Very nice, fixed it!